Projects/Server/Mobiles/Mobile.cs

```diff
@@ -3220,38 +3142,37 @@ public virtual void ProcessDelta()
                         ourState.Send(new HealthbarYellow(m));
                     }
                 }
-                else
-                {
-                    if (sendMoving || sendHealthbarPoison || sendHealthbarYellow)
-                    {
-                        var noto = Notoriety.Compute(m, m);
-                        ourState.Send(cache[1][noto] = Packet.Acquire(new MobileMovingOld(m, noto)));
-                    }
-                }
 
                 if (sendPublicStats || sendPrivateStats)
                 {
                     ourState.Send(new MobileStatusExtended(m, m_NetState));
                 }
                 else if (sendAll)
                 {
-                    ourState.Send(new MobileAttributes(m));
+                    Span<byte> attributesPacket = stackalloc byte[OutgoingMobilePackets.MobileAttributesPacketLength];
+                    OutgoingMobilePackets.CreateMobileAttributes(ref attributesPacket, m);
+                    ourState.Send(attributesPacket);
                 }
                 else if (sendAny)
                 {
+                    Span<byte> statsPacket = stackalloc byte[OutgoingMobilePackets.MobileStatsPacketLength];
```

Comment:
I have a theory that it is actually slower to reuse the same struct vs using 3 separate ones due to parallelization at the JIT/CPU level. Not really worth testing though.